src/components/tasks/TaskList/TaskList.tsx

```diff
@@ -49,6 +51,12 @@ export default function TaskList({
     updateCardContent(id, cardDetails);
   }
   
+  if (!tasks || !tasks?.length) {
```

Comment:
Use one of either conditions, both are not required.
```
if(!tasks){}
if(!tasks?.length) {}
```

Reply:
Give taskKey a defaultValue, if taskKey not passed and tasks are falsy. Then this might be returned.
```
<div>No undefined tasks found</div>
```

Reply:
I think it would be a better idea to have the specific taskKey, for example if the length of `in_progress` tasks is 0, it would say, "no in progress tasks found". Gives much better clarity